app/store/configureStore.js

```diff
@@ -10,21 +10,28 @@ const logger = createLogger({
   collapsed: false,
   logger: console,
   predicate: (getState, action) => true
-})
+});
 
-const createStoreWithMiddleware = applyMiddleware(
+let middleware = [
```

Comment:
Let's name this as `middlewares` to be more expressive.
(I know grammatically it's a compound, but adding an `s` make it more readable as an array)